tests/cpp/test_host_ir_stream_lowering.cpp

```diff
@@ -757,9 +795,11 @@ TEST_F(MultiDeviceExecutorLowerStreamTest, Matmul_N) {
   MultiDeviceExecutor executor(std::move(fusion), Communicator::getInstance());
 
   hir::HostIrContainer* container = executor.hostIrEvaluator()->container();
-  EXPECT_EQ(container->topLevelExprs().size(), 2);
+  EXPECT_EQ(container->topLevelExprs().size(), 4);
```

Comment:
FYI, https://github.com/NVIDIA/Fuser/pull/4427 will allow you to simplify the checks here.

Reply:
Thanks! That's useful. I will leave it like this for now but I'll keep that in mind and might change it later.